tests/attention/test_trtllm_gen_attention.py

```
@@ -1837,3 +1837,155 @@ def test_trtllm_batch_decode_spec(
+        cumsum_s_qo, num_qo_heads, head_dim_qk, device=device, dtype=torch.bfloat16
+    )
+    k_cache = torch.randn(
+        cumsum_s_kv, num_kv_heads, head_dim_qk, device=device, dtype=torch.bfloat16
+    )
+    v_cache = torch.randn(
+        cumsum_s_kv, num_kv_heads, head_dim_vo, device=device, dtype=torch.bfloat16
+    )
+
+    scale = float(1.0 / (head_dim_qk**0.5))
+
+    workspace_buffer, workspace_buffer_ref = create_workspace_buffers(device)
```

Ruff (0.15.9)

[warning] Unpacked variable `workspace_buffer_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)